synapse/tests/test_lib_stormhttp.py

```diff
@@ -46,27 +46,38 @@ async def test_storm_http_get(self):
 
             # Header and params as dict
             q = '''
-            $params=$lib.dict(key=valu, foo=bar)
-            $hdr = (
-                    ("User-Agent", "Storm HTTP Stuff"),
-            )
+            $params=$lib.dict(key=valu, foo=bar, baz=$lib.false)
+            $hdr = $lib.dict(true=$lib.true)
+            $hdr."User-Agent"="Storm HTTP Stuff"
+            $k = (0)
+            $hdr.$k="Why"
             $resp = $lib.inet.http.get($url, headers=$hdr, params=$params, ssl_verify=$lib.false)
             return ( $resp.json() )
             '''
             resp = await core.callStorm(q, opts=opts)
             data = resp.get('result')
-            self.eq(data.get('params'), {'key': ('valu',), 'foo': ('bar',)})
+            self.eq(data.get('params'), {'key': ('valu',), 'foo': ('bar',), 'baz': ('False',)})
```

Comment:
Do we wanna discuss the "python repr" vs more common JSON repr issue here at all?

Reply:
per discussion, deferring this question down the road.